src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -419,7 +421,8 @@ handle_info(random_promotion, #state{active=Active0,
                     State0;
                 false ->
                     % lager:info("Random promotion for node ~p", [Myself0]),
-                    move_random_peer_from_passive_to_active(State0)
+                    RandomPeer = select_random(Passive, [Myself]),
```

Comment:
Just to keep the same behaviour after the refactor which is selecting a random peer from the passive view to promote to the active one